README.md

```diff
@@ -205,8 +205,10 @@ Restful.js exposes similar methods on collections, members and entities. The nam
 * `patch ( id, data [, headers ] )`: Patch a member in a collection. Returns a promise with the response.
 * `head ( id, [, headers ] )`: Perform a HEAD request on a member in a collection. Returns a promise with the response.
 * `url ()`: Get the collection url.
-* `addResponseInterceptor ( interceptor )`: Add a response interceptor.
+* `addResponseInterceptor ( interceptor )`: Add a response interceptor. You can only alter data and headers.
 * `addRequestInterceptor ( interceptor )`: Add a request interceptor.
+* `addFullResponseInterceptor ( interceptor )`: Add a full response interceptor. You can alter data and headers.
```

Comment:
One interceptor is executed by `axios`, the other by the `restful.js` library, and the usage is clearer on the `restful.js` one, no magic modification. 
But in fact, there is no real difference, I wasn't see the comment in the documentation before starting the `fullResponseInterceptor`.